src/tests/integration/test-data/foundry-init-code-tests

```diff
@@ -0,0 +1,6 @@
+InitCodeTest.test_init()
```

Comment:
I know that's nitpicking, but to be consistent with other similar files I'd suggest to rename `foundry-init-code-tests`  to `foundry-init-code` or `foundry-init-code-all` (the latter is useful if there'll be a file with tests we'd like to skip such as `foundry-init-code-skip`).